monitors/CRMonitor.js

```diff
@@ -123,7 +125,7 @@ class CRMonitor {
   }
 
   _getPositionInformation(address) {
-    return this.empClient.getAllPositions().find(position => position.sponsor == address);
+    return this.empClient.getAllPositions().find(position => position.sponsor === address);
```

Comment:
Currently this line is failing to find the passed in address-to-monitor because it is not checksummed. I added a third equal sign also to be consistent with our existing style